talk/C++Course.tex

```diff
@@ -8,6 +8,9 @@
 \setboolean{onlybasics}{false}
 \IfFileExists{onlybasics.tex}{\input{onlybasics}}{}
 
+\newunicodechar{σ}{$\sigma$}
+\newunicodechar{µ}{$\mu$}
```

Comment:
Here I used `\DeclareUnicodeCharacter` initially, but this did not work for the CI.

Reply:
I'm not sure what you're playing with here. Why to define sigma or mu when it's already done for you.... Don't tell me it's the CI again !

Reply:
How is it already done for me? When I omit these lines I get:
![image](https://user-images.githubusercontent.com/1224051/191777748-e1d80389-df14-48cf-93d3-34a06f108a7d.png)